mythril/laser/ethereum/svm.py

```diff
@@ -154,17 +154,18 @@ def _get_covered_instructions(self) -> int:
             )
         return total_covered_instructions
 
-    def exec(self, create=False) -> None:
+    def exec(self, create=False, track_gas=False) -> None:
```

Comment:
Now the return type is incorrect

Reply:
Why is this resolved>

Reply:
The latest commits hadn't shown yet

Reply:
https://github.com/ConsenSys/mythril-classic/pull/660/commits/e47c7a35b268a3842342aa58b866cdf14d986b83
I updated the type hint in this commit